ios/Sources/AboutFeature/AboutDroidKaigiScreen.swift

```diff
@@ -0,0 +1,87 @@
+import SwiftUI
+import Styleguide
+
+public struct AboutDroidKaigiScreen: View {
+    @Environment(\.presentationMode) var presentationMode
+
+    public init() {
+        UITableView.appearance().backgroundColor = .clear
+        UINavigationBar.appearance().backgroundColor = .clear
+        UINavigationBar.appearance().barTintColor = .clear
+    }
+
+    public var body: some View {
+        NavigationView {
+            VStack {
+                VStack {
+                    Text(L10n.AboutDroidKaigiScreen.whatIs).font(.system(size: 22))
+                    AssetImage.logoTitle.image
+                        .frame(width: 261.81, height: 41.06, alignment: .center)
+                    Text(L10n.AboutDroidKaigiScreen.description)
+                        .multilineTextAlignment(.center)
+                        .font(.system(size: 17))
+                        .opacity(0.7)
+                }
+                .padding(.horizontal, 32)
```

Comment:
Spacing size looks different with Figma. In addition, use stack spacing could be better to read.

Reply:
change NavigationBarTitle configuration ```.large``` to ```.inline```, but NavigationBar's background Color has been untransparent😅

```Swift
            .introspectNavigationController { navigationController in
                navigationController.navigationBar.backgroundColor = .clear
            }
```

It doesn't work when building ...
<img src="https://user-images.githubusercontent.com/59905087/122349419-357f4580-cf87-11eb-9a6a-da95ce8fcf64.png" width="300" />

Reply:
@Tech-Nayuta 
Using `InlineTitleNavigationBarScrollView` might solve your problem. 
Please take a look 👍

Reply:
@ostk0069 
I tried Using ```InlineTitleNavigationBarScrollView```  on this screen and solved this problem, but ```ListView``` hasn't been displayed.  Do you know the cause that it hasn't been displayed? (I tried this pattern on SettingScreen and the same symptom appeared.) Should I use ```Vstack``` instead of using ```List```?

<img src="https://user-images.githubusercontent.com/59905087/122703414-75e30a00-d28c-11eb-973c-a8ad2c11da00.png" width="300" />

Reply:
@Tech-Nayuta
Thanks for trying. I also tried, but didn't work for navigation bar color.
For navigation color, let's keep using `UINavigationBar.appearance()` when initialized. 🙇